api/v1alpha1/limitador_types.go

```diff
@@ -115,8 +115,8 @@ type StorageType string
 
 const (
 	StorageTypeInMemory    StorageType = "memory"
-	StorageTypeRedis                   = "redis"
-	StorageTypeRedisCached             = "redis_cached"
+	StorageTypeRedis       StorageType = "redis"
```

Comment:
is this because of the linting task?

Reply:
yes, types should be explicit. Only using `iota` it is allowed type inference (which is `int`)

```go
const (
	C0 = iota
	C1
	C2
)
```